pipelines/artefact-automation-pipelines/promote-between-environments-after-approval/pipeline.yaml

```
@@ -0,0 +1,64 @@
+name: Approval before Promotion
+
+on:
+  artefact:
+    type: workload
+    include:
+    - myProject/myWorkloadArtefact
+    ref: refs/heads/main
```

this needs to be match-ref
```suggestion
    match-ref: refs/heads/main
```